dc-commons-xml/src/main/java/de/digitalcollections/commons/xml/xpath/UniversalNamespaceContext.java

```diff
@@ -19,7 +19,11 @@ public UniversalNamespaceContext(Document doc) {
     this.customPrefixes = new HashMap<>();
   }
 
-  /** Add a user-defined namespace. Takes precedence over the document's declared namespaces. */
+  /**
+   * Add a user-defined namespace. Takes precedence over the document's declared namespaces.
+   * @param prefix the prefix
```

Comment:
Braucht's solche Beschreibungen? Das ist IMO nur Noise, weil exakt dasselbe wie der Parametername.

Reply:
Leider bestand die javadoc-Validierung darauf. Die Alternative wäre, den ursprünglichen docstring in einen einfachen Kommentar umzuwandeln.

Reply:
wie besprochen...